cmd/job/submit/submit_mpi.go

```
@@ -82,6 +84,7 @@ func NewRunaiSubmitMPIJobCommand() *cobra.Command {
 	}
 
 	command.Flags().IntVar(&submitArgs.NumberProcesses, "processes", 1, "Number of distributed training processes.")
+	flags.AddBoolNullableFlag(command.Flags(), &(submitArgs.generateName), "generate-name", "", "Allow the CLI to change the name of the job if the job name already exists")
```

Maybe this is more suitable: 
`Add suffix to the job name if a job with the submitted name already exists`
@yarongol What do you think?

---

I don't have any context @yodarshafrir1  on what this pull request is about.
Why do we want to add a suffix to the job if it already exists? It can lead to confusion where you don't read the output and think that the job you created is the original job.
Why not just return an error "Job already exists, please use a different name or delete the original job"